virt_lightning/api.py

```diff
@@ -142,6 +142,16 @@ def _start_domain(hv, host, context, configuration):
                 network["network"], host["name"], ipaddress.ip_interface(ipv4)
             )
         domain.attach_network(**network)
+
+    disks = host.get("disks", [])
+    for i, disk in enumerate(disks):
+        size = 1
+        if "size" in disk and isinstance(disk["size"], int):
```

Comment:
If the user writes the size as a string, the value will be silently ignore. This can lead to confusion. I would suggest to either accept `str` too or relax the check to accept any value.